src/adb/index.js

```diff
@@ -0,0 +1,6 @@
+let fs = require('fs');
+
+fs.readdirSync(__dirname)
+  .filter(filename => !__filename.endsWith(filename))
+  .map(filename => filename.replace(/\.[^\.]+$/, ''))  // remove file extension
```

Comment:
This is just going from `logcat.js` to `logcat`, but I think that `path.basename(filename, '.js')` would work. Does that sound good?